src/reducers/historyReducer.js

```diff
@@ -0,0 +1,16 @@
+const historyReducer = (state, { type, payload }) => {
+  switch (type) {
+    case "INITIALISE_HISTORY":
+      return [...payload.items];
+    case "ADD_TO_HISTORY":
+      return [...payload.items];
+    case "REMOVE_FROM_HISTORY":
+      return [...payload.items];
+    case "REMOVE_USER_HISTORY":
+      return [...payload.items];
+    default:
```

Comment:
Is it mutating data? Looks like just returning what comes. Then why use reducers in the first place?